packages/int2lm/package.py

```diff
@@ -49,6 +49,7 @@ class Int2lm(MakefilePackage):
     depends_on('cosmo-eccodes-definitions',
                type=('build', 'link', 'run'),
                when='+eccodes')
+    depends_on('libaec', type=('build', 'link', 'run'), when='^eccodes +aec')
```

Comment:
Is this necessary, because it sounds wrong? I think int2lm doesn't depends on libaec. Int2lm depends on eccodes which depends on libaec. AFAIK indirect dependencies are inferred automatically.

Reply:
Yes I was wondering the same. I removed this line and int2lm was still building successfully.